src/pymodaq/utils/leco/daq_xDviewer_LECODirector.py

```diff
@@ -33,6 +35,7 @@ class DAQ_xDViewer_LECODirector(LECODirector, DAQ_Viewer_base):
 
     def __init__(self, parent=None, params_state=None, grabber_type: str = "0D", **kwargs) -> None:
         super().__init__(parent=parent, params_state=params_state, **kwargs)
+        self._register_DataFromPlugins_for_deserialization()
```

Comment:
if manual registration is still necessary, I suggest we do it in some __init__.py file of the corresponding package. Same for the base data objects

Reply:
Manual registration is only needed, if deserialization happens in another process.